### ISSUE TITLE
Open dropdown by default

### ISSUE THREAD

**Issue by MoBosse** [open-sdg/open-sdg#1349]
**Is your feature request related to a problem? Please describe.**
In several cases, we present indicators with multiple time series. Some of them have one series that complies with the global metadata and one or more series that provide additional information (see e.g. [Indicator 4.b.1](https://sdg-indikatoren.de/en/4-b-1/)). In these cases, only the compliant series should be a "start_value".
Now, we've had the case several times in the last few weeks where we've sent a link to an indicator to an external data provider so that they can look at how the data is presented, and they've reported back that they can't find the data on the platform. 

Obviously, the function of the dropdown selection to the left of the chart is not directly apparent to all users.

**Describe the solution that you need**
The selection of time series should be made more prominent.
Is there perhaps the possibility to have the dropdown selection already expanded when loading the page if there is a selection of time series?

**Describe alternatives you've considered**
Using the series toggle would make the selection of the series more obvious but is not allways an option since we want to display multiple series in one chart for comparison.

**Comment by LucyGwilliamAdmin** [comment]
@MoBosse have you considered using the radio button series instead e.g. https://sdgdata.gov.uk/17-3-1/

**Comment by MoBosse** [comment]
@LucyGwilliamAdmin Yes, we have considered this. From a purely visual point of view this is a good solution, but in many cases we do not want to give up the possibility to display the time series in the same chart/table for better comparability.
